samples/iree-reduce/run_reducer.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+iree-reduce -v interesting.sh softmax.mlir
```

Comment:
remove this script and add a README.md with this single command line instead

Reply:
Moved this as a filecheck RUN: command